pkg/loadbalancer/redirectpolicy/testdata/pod-readiness.txtar

```diff
@@ -0,0 +1,174 @@
+# Tests that pod with non-ready status are not added as LRP backends.
+
+hive/start
+
+# Add LRP, service and pods.
+k8s/add service.yaml lrp-svc.yaml lrp-addr.yaml pod-before.yaml
+# Give some time for reconciliation loop to run
+sleep 500ms
+
+# No local-redirect frontends with associated backends as pods are not "ready".
+db/cmp localredirectpolicies lrp.table
+db/cmp services services.table
+db/cmp frontends frontends-before.table
+
+# Update the pod status. The frontends should now be redirected.
+k8s/update pod.yaml
+db/cmp frontends frontends.table
+
+# -----
+
+-- lrp-svc.yaml --
+apiVersion: "cilium.io/v2"
+kind: CiliumLocalRedirectPolicy
+metadata:
+  name: "lrp-svc"
+  namespace: "test"
+spec:
+  redirectFrontend:
+    serviceMatcher:
+      serviceName: echo
+      namespace: test
+  redirectBackend:
+    localEndpointSelector:
+      matchLabels:
+        app: proxy
+    toPorts:
+      - port: "80"
+        name: "tcp"
+        protocol: TCP
+
+-- lrp-addr.yaml --
+apiVersion: "cilium.io/v2"
+kind: CiliumLocalRedirectPolicy
+metadata:
+  name: "lrp-addr"
+  namespace: "test"
+spec:
+  redirectFrontend:
+    addressMatcher:
+      ip: "169.254.169.254"
+      toPorts:
+        - port: "8080"
+          protocol: TCP
+  redirectBackend:
+    localEndpointSelector:
+      matchLabels:
+        app: proxy
+    toPorts:
+      - port: "80"
+        protocol: TCP
+
+-- pod-before.yaml --
```

Comment:
Could maybe just have single `pod.yaml` and use the `sed` command to flip the ready status (example sed use here: https://github.com/cilium/cilium/blob/main/pkg/loadbalancer/tests/testdata/reuse.txtar#L24)

Reply:
Done!